src/CLIUtils.ts

```diff
@@ -75,13 +77,67 @@ export function getHeadlessEnvVar(headless?: string) {
   }
 }
 
+type Result = {
+  startedAt: Date;
+  file: string;
+  finishedAt: Date;
+  passed: boolean;
```

Comment:
(nit): Since this library is not about testing but about "replay", wdyt of using a "result" field and using "Success" or "Failure" in the outputted text?

Reply:
Updated with `success` 🙏